migration-tools/utils.py

```diff
@@ -163,6 +163,16 @@ def migrate_docublock_output(exampleName):
 
     return output
 
+def migrateIndentedCodeblocks(paragraph):
+    indentedCodeblocks = re.findall(r"^\n(?:\s{4}[a-zA-Z].*\n)+^\n", paragraph, re.MULTILINE)
```

Comment:
`\s{4}` -> `[ ]{4}`, so we don't capture line breaks. There should also be no tab stops anywhere, but if there are any, it's a mistake.

Reply:
Shifted to using the commonmark parser lib in python to solve this issue, regex is not used anymore.